src/ctapipe/resources/stats_calc_config.yaml

```diff
@@ -0,0 +1,37 @@
+StatisticsCalculatorTool:
```

Comment:
Since we don't use `yaml` for anything apart from the configurations, I suggest to rename the configuration file to just `tool_name.yaml`, i.e. stripping `_config`.